lib/lib.rs

```diff
@@ -14,7 +14,7 @@ use deno_graph::source::LoadFuture;
 use deno_graph::source::LoadOptions;
 use deno_graph::source::Loader;
 use deno_graph::source::NullFileSystem;
-use deno_graph::source::ResolutionMode;
+use deno_graph::source::ResolutionKind;
```

Comment:
Renamed so that `ResolutionMode` aligns with TypeScript's wording https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-3.html#stable-support-resolution-mode-in-import-types